### ISSUE TITLE
Manage internationalization

### ISSUE THREAD

**Issue by ThomasG77** [kosmtik/kosmtik#42]
I'm interested by the topic and would like to contribute. I didn't see a reference to this feature in the code to start from something.
Do you have already ideas for managing it? I can make some search if not.

**Comment by yohanboniface** [comment]
If we are talking about the UI, I would go on https://github.com/yohanboniface/Leaflet.i18n because I know it suits my taste ;)
Not sure what would be the best way to make simple for plugins to inherit from this though. I need to think about it a bit.
About the backend, I'm not sure it needs internationalisation (eg. console outputs).

**Comment by ThomasG77** [comment]
Yes, only UI, end users in mind not devs: they are supposed to understand English when working IMO.

**Comment by yohanboniface** [comment]
The big picture would be something like:
- a way to compile every string to be translated from code base (maybe add a custom command, than uses Leaflet.I18n own command, but taking the kosmtik context into account to look for the right places without needing configuration)
- expose those strings to Transifex
- add a lang switcher somewhere on the UI (certainly on the top bar)
- make that i18n files are loaded/reloaded dynamically according to user lang choice
- define a sort of convention for plugins to be able to run the i18n command line and to expose their translated string (for example files go into `{plugin_root}/locale/{lang_code}.json`)

**Comment by kocio-pl** [comment]
If you still plan to do it, I will make a polish translation for sure. =}
